demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -222,3 +225,34 @@ def test_IsAskConditionHasUnhandledReplyOptionsValidator():
         )
     }
     assert IsAskConditionHasUnhandledReplyOptionsValidator().is_valid([playbook])
+
+
+def test_IsTaskidDifferentFromidValidator():
+    playbook = create_playbook_object()
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test",
+                "taskid": "test",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+            }
+        )
+    }
```

Comment:
You can remove this part, the object returned from create_playbook_object() should contain tasks and be valid already.